docs/content/en/docs/getting-started/metrics/_index.md

```diff
@@ -4,3 +4,321 @@ description: Learn how Keptn metrics enhances your deployment
+        keptn.sh/post-deployment-tasks: notify
+...
+```
+
+For more information about using annotations and labels
+to integrate KLT into your deployment cluster, see
+[Integrate KLT with your applications](../../implementing/integrate.md).
+
+The [Kubernetes metric server](https://github.com/kubernetes-sigs/metrics-server)
+requires that you maintain point-to-point integrations
+from Argo Rollouts, Flux, KEDA, and HPA.
+Each has plugins but it is difficult to maintain them,
+especially if you are using multiple tools
+and multible observability platforms.
+The Keptn Metrics Server unifies and standardizes access to this data.
+
+The steps to implement Keptn metrics are:
+
+## Install and configure KLT
+
+Use the Helm Chart to install the Keptn Metrics Server
+as part of the Lifecycle Toolkit
+or completely stand-alone.
+ See
+[Install KLT using the Helm Chart](../../install/install.md/#use-helm-chart).
+-- End of Probably goes --
+
+## Define metrics to use
+
+You need to define the external observability platforms
+from which you want to pull data
+and then the specific data you want to pull.
+This data is pulled and fetched continuously
+at an interval you specify for each specific bit of data.
+Data is available through the resource and through the data provider itself,
+as well as the Kubernetes CLI.
+
+### Define metrics providers
+
+Populate a
+[KeptnMetricsProvider](../../yaml-crd-ref/metricsprovider.md)
+resource for each external observability platform you want to use.
+
+For our example, we define two observability platforms:
+
+* `dev-prometheus`
+* `dev-dynatrace`
+
+You can specify a virtually unlimited number of providers,
+including multiple instances of each observability platform.
+Each one must be assigned a unique name,
+identified by the type of platform it is
+and the URL.
+
+> Note: The video and example application use an older syntax
+  of the `KeptnMetricsProvider` and `KeptnMetric` resources.
+  The syntax shown in this document is correct for v0.7.1 and later.
+
+Definition of
+[dev-prometheus](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/keptn-prometheus-provider.yaml)
+data source:
+
+```yaml
+kind: KeptnMetricsProvider
+metadata:
+  name: dev-prometheus
+  namespace: simplenode-dev
+spec:
+  type: prometheus
+  targetserver: "http://prometheus-k8s-monitoring-svc.cluster.local:9090"
+```
+
+Definition of the
+[dev-dynatrace](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/dynatrace-provider.yaml.tmp)
+data source.
+Note that the `dev-dynatrace` server is protected by a secret key
+so that information is included in the provider definition:
+
+```yaml
+kind: KeptnMetricsProvider
+metadata:
+  name: dev-dynatrace
+  namespace: simplenode-dev
+spec:
+  type: dynatrace
+  targetServer: "https://hci34192.live.dynatrace.com
+  secretKeyRef
+    name: dynatrace
+    key: DT_TOKEN
+...
+```
+
```

Comment:
The secret key contains the API token to authenticate with your Dynatrace tenant. You can provide your own token by creating a [Kubernetes Secret](https://kubernetes.io/docs/concepts/configuration/secret/) like the following:
```yaml
apiVersion: v1
kind: Secret
metadata:
  name: dynatrace
  namespace: simplenode-dev
type: Opaque
data:
  DT_TOKEN: "<api-token>"
```

Reply:
I don't think this belongs in a Getting Started Guide but it belongs somewhere -- then the GSG can link to that other doc for people who want to learn about it.  We have a bit of different information about secrets that is currently squirreled away on the KeptnTaskDefinition (along with info about context) -- it doesn't belong there but that is where it was in the old docs so I left it for now.  I created https://github.com/keptn/lifecycle-toolkit/issues/1390 so we remember to do it.